sycl/test-e2e/Basic/queue/release.cpp

```diff
@@ -2,6 +2,7 @@
 // RUN: env SYCL_UR_TRACE=2 %{run} %t.out | FileCheck %s %if !windows %{--check-prefixes=CHECK-RELEASE%}
 //
 // XFAIL: hip_nvidia
+// XFAIL-TRACKER: https://github.com/intel/llvm/issues/16197
```

Comment:
Shouldn't we remove `XFAIL` instead?

Reply:
> Shouldn't we remove `XFAIL` instead?

Yes, this one slipped through the cracks. Fixed it now!